### PR TITLE
[STAN-235] UI: browse index changes

### PR DESCRIPTION
**Pull request by roc**
* Removes "all" from results copy
* Updates description text for browse index
* Feedback Footer updated

Screenshot 2022-02-15 at 13 04 46
Screenshot 2022-02-15 at 13 04 43

### CODE REVIEW

File: ui/pages/standards/index.js [hunk 1]
```diff
@@ -17,9 +18,18 @@ export default function Standards({ data, schemaData }) {
...
         <p>
-          Explore listings for information standards, services and APIs used in
-          health and social care technology. All of the standards listed in the
-          directory are nationally recognised.
+          Find standards, services and APIs to build interoperable technology in
+          health and social care.
+        </p>
+        <p>
+          To view, share and discuss standards use cases and local
+          specifications, go to the{' '}
```
Comment (lennym) [inline ui/pages/standards/index.js:26]: 🤮

**Comment by iebarat** [comment]
@roc how come it's showing 144 results? There should be only about 80+ in all, as the rest are set in private on CKAN. Is it just because it's in your local host and not connected to CKAN?

Btw we should not include the last line in the updated description as we haven't gone live with the GitHub community yet

File: ui/pages/standards/index.js (hunk 1 above)
Comment (roc) [reply ui/pages/standards/index.js:26]: Are you okay m8?

**Comment by roc** [comment]
> @roc how come it's showing 144 results? There should be only about 80+ in all, as the rest are set in private on CKAN. Is it just because it's in your local host and not connected to CKAN?

Hawk eyed! It's because the screenshots are from my local version of the site, which has 144 records.

> Btw we should not include the last line in the updated description as we haven't gone live with the GitHub community yet

Oft! Okay will remove.

File: ui/pages/standards/index.js (hunk 1 above)
Comment (lennym) [reply ui/pages/standards/index.js]: `No{' '}`

**Review by iebarat — APPROVED** [review]

File: .github/workflows/build-and-test.yml [hunk 2]
```diff
@@ -36,7 +35,7 @@ jobs:
           - 16.x
     env:
       PORT: 3000
-      CKAN_URL: http://a864b7b77f8e140858ab710899b7ed73-1561736528.eu-west-2.elb.amazonaws.com:5000/api/3/action
+      CKAN_URL: https://manage.dev.nhs.marvell-consulting.com/api/3/action
```
Comment (lennym) [inline .github/workflows/build-and-test.yml:38]: 😍

File: ui/pages/standards/index.js (hunk 1 above)
Comment (roc) [reply ui/pages/standards/index.js]: I've removed the entire paragraph now`{' '}`.

**Review by lennym — APPROVED** [review]

**Pull request merged**